basics/uri.adoc

```diff
@@ -330,99 +565,424 @@ Remote UUris mean that the uThing is running on a different device and we need t
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |        UENTITY_ID             |  UE_VERSION   |   UNUSED      |
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|    ID_LEN     |        UAUTHORITY_ID (1=256 bytes)  ...       |
+|   NAME_LEN    |        UAUTHORITY_NAME  ...                   |
 |                                                               |
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 ----
 
-== URI Examples
+[%autowidth]
+|===
+|Field |Description
+
+|TYPE
+a| The type of authority contained in this micro URI. Occupies bits 8 through 15 (octet 1). +
+*MUST* be set to `0x03` for a URI with an authority.
 
-The following examples are based off the USubscription notification topic (see usubscription.proto).
+|NAME_LEN
+a|The length of the `UAUTHORITY_NAME`. Occupies bits 64 through 71 (octet 24). +
+*MUST* contain the length of the `UAUTHORITY_NAME` field in bytes.
 
+|UAUTHORITY_NAME
```

Comment:
@sophokles73, This is simpler in that there is only local vs remote however it means that a 4 byte IPv4 address could take up to 15 bytes and the IPv6 16 bytes could take up to 39 bytes.

Reply:
I just wanted to point out that you **can** also use IP address strings for the name. But our overall idea was to suggest that uEntities use logical names only for the authority ...